examples/graph_kernels.pct.py

```diff
@@ -85,9 +85,9 @@
 
 true_kernel = gpx.GraphKernel(
     laplacian=L,
-    lengthscale=jnp.array([2.3]),
-    variance=jnp.array([3.2]),
-    smoothness=jnp.array([6.1]),
+    lengthscale=jnp.array(2.3),
+    variance=jnp.array(3.2),
+    smoothness=jnp.array(6.1),
```

Comment:
NB- should it be possible as a user to simply pass numpy floats, e.g.
```suggestion
    lengthscale=2.3,
    variance=3.2,
    smoothness=6.1,
```
?